From fbbeb01162a59e198acc89f5330e6fe7d661c4d4 Mon Sep 17 00:00:00 2001
From: Marius Gripsgard <marius@ubports.com>
Date: Tue, 10 Apr 2018 01:48:14 +0200
Subject: [PATCH] Fix pkgconf for debian based systems.

Since debian based systems uses lib/{arch}/ as libdir and /usr/lib/pulse-* for modules this causes -L to search for -ldroid-utils in lib/{arch}/
---
 src/common/libdroid-util.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/libdroid-util.pc.in b/src/common/libdroid-util.pc.in
index 523e4b0..32c77a1 100644
--- a/src/common/libdroid-util.pc.in
+++ b/src/common/libdroid-util.pc.in
@@ -7,5 +7,5 @@ libexecdir=@libexecdir@
 Name: libdroid-util
 Description: Common droid module building interface.
 Version: @PA_MODULE_VERSION@
-Libs: -L${libdir}/pulse-@PA_MAJORMINOR@/modules -ldroid-util
+Libs: -L${prefix}/lib/pulse-@PA_MAJORMINOR@/modules -ldroid-util
 Cflags: -D_REENTRANT -I${includedir}/pulsecore/modules
